app/Jobs/ImportFile.php

```diff
@@ -59,17 +59,6 @@ public function __construct($filepath, $importType, $importOptions)
         $this->importOptions = $importOptions;
     }
 
-    /**
-     * Get the tags that should be assigned to the job.
-     * TODO: Is this used anywhere?
```

Comment:
I don't think we [use this](https://github.com/DoSomething/chompy/pull/2#discussion_r178354112).